### DISCUSSION TITLE
3.01版本天翼网盘超过2G的文件就无法下载和播放

### DISCUSSION THREAD

**Discussion by beizi1754** [libsgh/PanIndex#251] [Q&A]
天翼网盘只要文件大小小于1G，不管是文件还是视频都可以正常下载或者播放。
但是超过2G的文件，点击下载会提示：404 page not found
不知道是天翼云的限制还是。。。

**Comment by beizi1754** [comment]
无论用API直连或完全缓存都是一样

**Reply by libsgh** [reply]
> 无论用API直连或完全缓存都是一样

更新到3.0.3，解决了接口限制问题

**Reply by beizi1754** [reply]
收到
